reverseCase/reverseCase.js

```diff
@@ -4,4 +4,27 @@
  * Assume that each string will contain only spaces and letters.
  */
 
```

Comment:
Good job man, this looks nice and readable. I love code that i can read and sift through the logic easily.